Source/Hurl.Selector/MainWindow.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using WinUIEx;
+
+namespace Hurl.Selector;
+
+internal class MainWindow
+{
+    Window? window;
+    WindowManager? windowManager;
+
+    public void CreateWindow()
+    {
+        if (window == null)
+        {
+            window = new Window()
+            {
+                SystemBackdrop = new MicaBackdrop(),
```

Comment:
Mica is originally intended for long(er) live windows. I think acrylic is good. Also this can be set in XAML.